Public/New-PolarisStaticRoute.ps1

```diff
@@ -110,14 +114,77 @@ function New-PolarisStaticRoute {
         }
     }
 
+    
     # Inserting variables into scriptblock as hardcoded
     $Scriptblock = [scriptblock]::Create(
         "`$RoutePath = '$($RoutePath.TrimStart("/"))'`r`n" +
         "`$EnableDirectoryBrowser = `$$EnableDirectoryBrowser`r`n" +
         "`$NewDrive = '$NewDrive'`r`n" +
```

Comment:
If you want to make a $StandardHTMLFiles parameter available in the scriptblock just insert it in here like:
```suggestion
        "`$NewDrive = '$NewDrive'`r`n" +
        "`$StandardHTMLFiles = @($($StandardHTMLFiles -join ','))`r`n" +
```